app/src/main/kotlin/org/fossify/calculator/helpers/converters/VolumeConverter.kt

```diff
@@ -187,6 +194,7 @@ object VolumeConverter : Converter {
         Unit.QuartUS,
         Unit.PintUS,
         Unit.GillUS,
+        Unit.CupUS,
```

Comment:
```suggestion
        Unit.CupUS,
        Unit.GillUS,
```

1 cup = 2 gills, so it should be earlier.